project_generator/commands/list_projects.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2014-2015 0xc0170
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from ..workspace import PgenWorkspace
+from ..util import unicode_available
+
+help = 'List all projects'
```

Comment:
Yes, let's do  that !

There seems to be problem with projects it prints None. Buggggggg!